modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/resource/test/HeadlessBuilderResourceTest.java

```diff
@@ -38,56 +37,25 @@
 
 import java.nio.charset.StandardCharsets;
 
-import java.util.Objects;
-import java.util.concurrent.CompletableFuture;
-
 import javax.ws.rs.core.HttpHeaders;
 
 import org.junit.After;
 import org.junit.Assert;
-import org.junit.Before;
 import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import org.osgi.framework.Bundle;
-import org.osgi.framework.BundleContext;
-import org.osgi.framework.FrameworkUtil;
 
 /**
  * @author Luis Miguel Barcos
  */
 @FeatureFlags({"LPS-186757", "LPS-184413", "LPS-167253", "LPS-153117"})
-@RunWith(Arquillian.class)
-public class HeadlessBuilderResourceTest {
+public class HeadlessBuilderResourceTest extends BaseTestCase {
 
 	@ClassRule
 	@Rule
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
-	@Before
-	public void setUp() {
-		Bundle testBundle = FrameworkUtil.getBundle(
-			APIApplicationProviderTest.class);
-
-		BundleContext bundleContext = testBundle.getBundleContext();
-
-		for (Bundle bundle : bundleContext.getBundles()) {
-			if (Objects.equals(
-					bundle.getSymbolicName(),
-					"com.liferay.headless.builder.impl")) {
-
-				CompletableFuture<Void> completableFuture =
-					_batchEngineUnitProcessor.processBatchEngineUnits(
-						_batchEngineUnitReader.getBatchEngineUnits(bundle));
-
-				completableFuture.join();
-			}
-		}
-	}
-
 	@After
 	public void tearDown() throws Exception {
```

Comment:
I think this is no longer needed either

Reply:
Done adapting the code to the current master